### DISCUSSION TITLE
Cannot load model.pb

### DISCUSSION THREAD

**Discussion by r-hou** [deepmodeling/deepmd-kit#197] [Q&A] [answered]
I installed deepmd-kit completely following the instructions on the main page. However, deepmd-kit cannot load a frozen model and gives the error message as follows:
tensorflow.python.framework.errors_impl.NotFoundError: Op type not registered 'DescrptNorot' in binary running on dirac.phys.ust.hk. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.

**Comment by njzjz** [comment, marked as answer]
Hi,
I guess that’s because the deeper-kit version you used to freeze the model and that you used to load the model are different.

————————
Jinzhe Zeng
PhD student in Chemistry
Rutgers University

jinzhe.zeng@rutgers.edu
CV: https://njzjz.win/cv
